django_project/frontend/src/containers/MainPage/SideBar/Filter.tsx

```diff
@@ -145,10 +137,55 @@ function Filter() {
         })
     }
 
+    const fetchPropertyList = () => {
+        setLoading(true)
+        axios.get(FETCH_PROPERTY_LIST_URL).then((response) => {
+            setLoading(false)
+            if (response.data) {
+                setPropertyList(response.data as PropertyInterface[])
+            }
+        }).catch((error) => {
+            setLoading(false)
+            console.log(error)
+        })
+    }
+
     useEffect(() => {
-        fetchSpeciesList()
+        fetchSpeciesList();
+        fetchPropertyList();
     }, [])
 
+    const handleDeleteSpecies = (valueToDelete: string) => {
+        const updatedSelectedSpecies = selectedSpecies.filter((item) => item !== valueToDelete);
+        setSelectedSpecies(updatedSelectedSpecies);
+        dispatch(toggleSpecies(updatedSelectedSpecies.length > 0 ? updatedSelectedSpecies.join(',') : ''));
```

Comment:
Please avoid repeating yourself multiple times. You can create a `useEffect` for `selectedSpecies` and then run `dispatch` within it.
```
useEffect(() => {
    dispatch(toggleSpecies(selectedSpecies.length > 0 ? selectedSpecies.join(',') : ''));
}, [selectedSpecies])
```